CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## unreleased
 * Re-add `BraintreeCore` dependency to `PayPalDataCollector` for Swift Package Manager archive issue workaround (fixes #679)
+* Remove SPM product libraries for `KountDataCollector` and `PPRiskMagnes` (this was a workaround for an Xcode bug discussed in #576; bug resolved in Xcode 12.5)
```

Comment:
```suggestion
* Swift Package Manager
    * Re-add `BraintreeCore` dependency to `PayPalDataCollector` for Swift Package Manager archive issue workaround (fixes #679)
    * Remove SPM product libraries for `KountDataCollector` and `PPRiskMagnes` (this was a workaround for an Xcode bug discussed in #576; bug resolved in Xcode 12.5)
```

Reply:
Might be nice to group both of these under `Swift Package Manager`.

Also, maybe we should add a note in the changelog calling out that merchants using SPM can delete the Kount run script and remove the explicit dependencies on `KountDataCollector` and `PPRiskMagnes`?

And shouldn't `CardinalMobile` be listed here too?

Reply:
Good callouts - I made an update, let me know what you think!